src/components/forms/ClaimForm.svelte

```diff
@@ -167,8 +168,10 @@ function onInfoClick(event: Event) {
 </script>
 
 <div class="w-50">
-  {#if !hideItemName}
+  {#if items.length < 1}
```

Comment:
Seems this is more conventional, but my personal preference would be `if items.length > 0` and reverse the conditions. Go with your own preference, though.
```suggestion
  {#if !items?.length}
```